neat-dnfs/analysis/viz/genome.py

```
@@ -0,0 +1,400 @@
+    lab = get_element_label(k_el)
+    lab_lower = lab.lower()
+
+    if "mexican" in lab_lower:
+        Ae = k_el.get("amplitudeExc")
+        se = k_el.get("widthExc")
+        Ai = k_el.get("amplitudeInh")
+        si = k_el.get("widthInh")
+        Ag = k_el.get("amplitudeGlobal", 0.0)
+        return (
+            "Mexican-hat kernel: "
+            f"A_exc = {Ae:.2f}, σ_exc = {se:.2f}, "
```

Ruff (0.16.1)

[warning] String contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)